signatures/Picnic/Picnic-FS/param/L1-FS.yaml

```diff
@@ -0,0 +1,13 @@
+name: L1-FS
+security level:  
+  classical: 128
+  quantum:  64
+  nist category: 1
+failure probability: 0
+number of operations: inf
+sizes:
+  sk: 16
+  pk: 32
+  ct|sig: 34032
+  msg: 256
```

Comment:
I think they use the message unmodified (they only say it must be shorter than 2^55 bytes) (page 10, § 6.2)

So I'd just leave this out.

Reply:
I would suggest to right that as a comment  message is shorter than 2^55 bytes. what do you think?